folsom/src/main/java/com/spotify/folsom/client/ascii/DefaultAsciiMemcacheClient.java

```diff
@@ -161,11 +185,20 @@ public DefaultAsciiMemcacheClient(
 
   @Override
   public CompletionStage<MemcacheStatus> replace(String key, V value, int ttl) {
+    return replace(key, value, ttl, 0);
```

Comment:
If we do my other suggestion about making these interface methods `default`, we should instead delegate to an internal private method to avoid infinite recursion

Reply:
Sorry, I'm a bit confused about this one—wouldn't it be safe to make this call here since `DefaultAsciiMemcacheClient` overloads the default implementation?

Reply:
Hm, yes I think you are right.